### DISCUSSION TITLE
Should OpenMP parallelization be disabled in libraries (ex blas) if compiling Mumps with OpenMP

### DISCUSSION THREAD

**Discussion by trevorqn** [scivision/mumps-superbuild#35] [General]
I'm not sure if this is the appropriate place to ask. But I'm compiling mumps with OpenMP and OpenBLAS and am wondering if one should use a serial blas library, to avoid nested parallellism (which could lead to oversubscribing cores), or if compiling everything for OpenMP should be fine?

**Comment by scivision** [comment]
This would be for the [MUMPS users list](https://listes.ens-lyon.fr/sympa/subscribe/mumps-users) thanks for reaching out
